meta-dts-distro/recipes-dts/dts/dts.bb

```diff
@@ -13,7 +13,7 @@ SRC_URI = " \
     file://cloud_list \
     file://dts-environment.sh \
     file://dts-functions.sh \
-    file://coreboot_customizer \
+    gitsm://github.com/Dasharo/dcu.git;protocol=https;nobranch=1 \
```

Comment:
@filipleple Not like this, we should create a separate recipe for this. I will handle that.